.packit.yaml

```diff
@@ -38,3 +38,20 @@ jobs:
       # do not get the version from a tag (git describe) but from the spec file
       get-current-version:
         - grep -oP '^Version:\s+\K\S+' packaging/command-line-assistant.spec
+  - job: tests
+    trigger: pull_request
+    fmf_url: "https://gitlab.cee.redhat.com/rhel-lightspeed/enhanced-shell/cla-tests"
```

Comment:
until made public, we can try this
https://github.com/danmyway/shellai-tests

Reply:
done